examples/with-next-css/next.config.js

```diff
@@ -0,0 +1,26 @@
+const withCSS = require('@zeit/next-css')
+/* Without CSS Modules, with PostCSS */
+module.exports = withCSS()
+
+/* With CSS Modules */
+// module.exports = withCSS({ cssModules: true })
+
+/* With additional configuration on top of CSS Modules
+In this case I have added the WebpackBundleAnalyzer plugin, as is demonstrated in this example, https://github.com/zeit/next.js/tree/canary/examples/with-webpack-bundle-analyzer
+*/
+/*
+const { ANALYZE } = process.env
+const { WebpackBundleSizeAnalyzerPlugin } = require('webpack-bundle-size-analyzer')
+
+module.exports = withCSS({
+  cssModules: true,
+  webpack: function (config) {
+    if (ANALYZE) {
```

Comment:
Lets remove this. It's not related to the example. It can just be `return config`